lib/prm/declaration/api.ex

```diff
@@ -54,4 +62,33 @@ defmodule PRM.DeclarationAPI do
     declaration
     |> cast(attrs, fields)
   end
+
+  defp declaration_search_changeset(%Declaration{} = declaration, attrs) do
+    fields = [
+      :patient_id,
+      :active,
+      :doctor_id,
+      :msp_id
+    ]
+
+    declaration
+    |> cast(attrs, fields)
+    |> validate_any_is_present(fields)
+  end
+
+  defp validate_any_is_present(changeset, fields) do
```

Comment:
Maybe lets move it to `EView.Changeset.Validator*`? We already have common validators extensions there and looks like we will re-use this thing over and over again.